GitVersionCore/GitVersionContext.cs

```diff
@@ -9,6 +11,8 @@
     /// </summary>
     public class GitVersionContext
     {
+        readonly bool IsContextForTrackedBranchesOnly;
```

Comment:
No idea what this field is, but is it correct that the default changed from true to false?